inc/admin/dashboard/main.php

```diff
@@ -43,6 +43,7 @@ class Main {
 	 */
 	private $useful_plugins = [
 		'optimole-wp',
+		'wplk',
```

Comment:
Let's replace this with the actual plugin slug, maybe we'll use it in the future.